iterators.tex

```diff
@@ -276,7 +276,7 @@
   template <class I>
   concept bool Readable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<I>@\newtxt{()}@ &&
-      requires (I i) {
+      requires (@\newtxt{const }@I i) {
```

Comment:
Minor discomfort. An `optional` is Readable, and it has both const and non-const `operator*` overloads.